mlflow/sklearn/__init__.py

```diff
@@ -531,6 +531,30 @@ def autolog():
       **Metrics**
         - A training score obtained by ``estimator.score``. Note that the training score is
           computed using parameters given to ``fit()``.
+        - Common metrics for classifier:
+          .. _precision score:
+              https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html
+          .. _recall score:
+              https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html
+          .. _f1 score:
+              https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html
+          .. _accuracy score:
+              https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html
+
+          If the classifier has method ``predict_proba``, we additionally log:
+
+          .. _log loss:
+              https://scikit-learn.org/stable/modules/generated/sklearn.metrics.log_loss.html
+          .. _roc auc score:
+              https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html
```

Comment:
Thanks for the fix, but just defining link targets doesn't generate links.

<img width="799" alt="Screen Shot 2020-09-01 at 16 46 00" src="https://user-images.githubusercontent.com/17039389/91822346-a365b500-ec72-11ea-9822-c091a396ad7b.png">

https://927-136202695-gh.circle-artifacts.com/0/docs/build/html/python_api/mlflow.sklearn.html

Reply:
You can verify the doc is rendered properly by following the instruction here: https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.rst#writing-docs

Reply:
you also verify the doc on CircleCI:

![Sep-01-2020 16-55-20](https://user-images.githubusercontent.com/17039389/91823401-21768b80-ec74-11ea-8098-5c0105372019.gif)

Reply:
Gotcha. Before I thought this is just a pure comment and haven't realized that this comment is for documentation lol.